src/main/java/com/epam/lowcost/controller/UserController.java

```diff
@@ -125,10 +126,16 @@ public String changePassword(@ModelAttribute("sessionUser") User sessionUser, @R
     }
 
     @PostMapping(value = DELETE)
-    public String deleteUser(@ModelAttribute(value = "sessionUser") User sessionUser, @RequestParam long id, Model model) {
+    public String deleteUser(@RequestParam long id, ModelMap model) {
+        User sessionUser = (User) model.get("sessionUser");
         if (!sessionUser.isAdmin()) {
             return "redirect:" + TICKETS + SELF;
         }
+        if (sessionUser.getId() == id) {
+            System.out.println(sessionUser.getId() + "" + id);
+            model.addAttribute("message", "You cant delete yourself!");
+            return "users";
```

Comment:
In me opinion all names of views should be as constants too. single new task

Reply:
Made a card in trello.